azurelinuxagent/common/osutil/nsbsd.py

```diff
@@ -153,3 +152,15 @@ def remove_firewall(self, dst_ip=None, uid=None):
     def enable_firewall(self, dst_ip=None, uid=None):
         # disable iptables methods
         return True
+
+    def supports_logrotate(self):
+        return False
+
+    def get_agent_log_dir(self):
+        return "/log/azure"
+
+    def get_agent_log_rotation_maxbytes(self):
+        return 4000000
```

Comment:
Its the configuration that @remip2 put it. I think it was because of the limited space available on the disks for `nsbsd` systems.

@remip2 Can you clarify this better?

Reply:
Yes, 4Mo is just the max size before rotation for all the other log files in nsbsd system. I just want to keep it consistent for nsbsd.